src/mod_push_service_mongoosepush.erl

```diff
@@ -79,12 +79,18 @@ config_spec() ->
 %% Hooks
 %%--------------------------------------------------------------------
 
+-spec hooks(mongooseim:host_type()) -> gen_hook:hook_list().
+hooks(HostType) ->
+    [{push_notifications, HostType, fun ?MODULE:push_notifications/3, #{}, 10}].
+
 %% Hook 'push_notifications'
--spec push_notifications(AccIn :: ok | mongoose_acc:t(), Host :: jid:server(),
-                         Notifications :: [#{binary() => binary()}],
-                         Options :: #{binary() => binary()}) ->
-    ok | {error, Reason :: term()}.
-push_notifications(AccIn, Host, Notifications, Options = #{<<"device_id">> := DeviceId}) ->
+-spec push_notifications(Acc, Params, Extra) -> {ok, ok | {error, Reason :: term()}} when
+    Acc :: ok | mongoose_acc:t(),
```

Comment:
That's the strangest accumulator, this hook probably needs some rework, but that's out of the scope of this PR